core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -138,11 +142,11 @@ object Boot extends App with LazyLogging {
     val submissionTimeout = util.toScalaDuration(executionServiceConfig.getDuration("workflowSubmissionTimeout"))
 
     val executionServiceServers: Map[ExecutionServiceId, ExecutionServiceDAO] = executionServiceConfig.getObject("readServers").map {
-        case (strName, strHostname) => (ExecutionServiceId(strName)->new HttpExecutionServiceDAO(strHostname.unwrapped.toString, submissionTimeout, metricsPrefix))
+        case (strName, strHostname) => (ExecutionServiceId(strName)->new HttpExecutionServiceDAO(strHostname.unwrapped.toString, metricsPrefix))
```

Comment:
submissionTimeout was from a day where it could take some time to submit to cromwell. It was too hard to reimplement in akka http client so I ripped it out. I did confirm via our instrumentation that our requests are sub second.

Reply:
additional context: IIRC it was when we had batch workflow submit, and Cromwell would process all the workflows before doing its HTTP response

Reply:
also true